ui/desktop/app/components/session/tabs/index.js

```diff
@@ -50,12 +50,29 @@ export default class SessionTerminalTabsComponent extends Component {
       return;
     }
 
+    const { isWindows } = await this.ipc.invoke('checkOS');
     const xterm = new Terminal(terminalOptions);
     this.terminal = xterm;
     const fitAddon = new FitAddon();
     const termContainer = document.getElementById(`terminal-container`);
     calculateTerminalContainerHeight(termContainer);
 
+    /**
+     * Custom key event handler for windows to achieve Ctrl+C as on the other OS's
+     */
+    if (isWindows) {
+      xterm.attachCustomKeyEventHandler((arg) => {
+        if (arg.ctrlKey && arg.code === 'KeyC' && arg.type === 'keydown') {
+          const selection = xterm.getSelection();
+          if (selection) {
+            navigator.clipboard.writeText(selection);
```

Comment:
non-blocking question for learning: Since `writeText` returns a promise should we use `await`OR in this case do we not use `await` since `writeText` is inside an event handler OR do we not want to use `await` since we are handling a user action and we want to return immediately?

Reply:
@lisbet-alvarez Good catch! Didn't though about handling the promise since I expected this handler to return immediately (so writing on the clipboard should be instantly). But thinking twice, I think, is a good idea, just in case it takes longer than expected to write on the clipboard.